metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1049,10 +1053,12 @@ class MetalsLanguageServer(
     // Update in-memory buffer contents from LSP client
     buffers.put(path, params.getTextDocument.getText)
 
-    packageProvider
-      .workspaceEdit(path)
-      .map(new ApplyWorkspaceEditParams(_))
-      .foreach(languageClient.applyEdit)
+    timerProvider.timedThunk("package", true) {
```

Comment:
Is this needed still? Second parameter is anyway true
```suggestion
    timerProvider.timedThunk("package") {
```

Reply:
removed